logging/logging_alerts/default-logging-alerts.adoc

```diff
@@ -0,0 +1,16 @@
+:_content-type: ASSEMBLY
+[id="default-logging-alerts"]
+include::_attributes/common-attributes.adoc[]
+= Default logging alerts
+:context: default-logging-alerts
+
+toc::[]
+
+Logging alerts are installed as part of the Cluster Logging Operator installation. These alerts depend on the collection of metrics, which is enabled if you selected the option to *Enable operator recommended cluster monitoring on this namespace* when installing the Cluster Logging Operator. For more information about installing logging Operators, see xref:../../logging/cluster-logging-deploying#cluster-logging-deploy-console_cluster-logging-deploying[Installing the {logging-title} using the web console].
+
+Application, audit, and infrastructure alerts are sent by default to the {product-title} monitoring stack Alertmanager in the `openshift-monitoring` namespace, unless you have disabled the local Alertmanager instance. However, if the Alertmanager that is used to monitor user-defined projects in the `openshift-user-workload-monitoring` namespace is enabled, application logs are sent here by default.
```

Comment:
Since here we are talking about alerts that users are not able to customize I feel we don't need to make the distinction between the 3 log types. We should instead just mention that these alerts are sent to `{product-title} monitoring stack Alertmanager` and move some of this info to `Custom logging alerts` where I feel it's more relevant.
```suggestion
These alerts are sent by default to the {product-title} monitoring stack Alertmanager in the `openshift-monitoring` namespace, unless you have disabled the local Alertmanager instance.
```